modules/motions/ui/MotionFormStartNew/Parts/StartNewSDVTVettedValidatorsLimitsSet.tsx

```diff
@@ -0,0 +1,201 @@
+      'trustedCaller',
+      [],
+    )
+
+    const fieldsArr = useFieldArray({ name: fieldNames.nodeOperators })
+    const { watch } = useFormContext()
+    const { isValid } = useFormState()
+    const selectedNodeOperators: NodeOperator[] = watch(
+      fieldNames.nodeOperators,
+    )
+
+    const getFilteredOptions = (fieldIdx: number) => {
+      if (!nodeOperatorsWithValidators?.length) return []
+      const selectedIds = selectedNodeOperators.map(({ id }) => parseInt(id))
+      const thisId = parseInt(selectedNodeOperators[fieldIdx]?.id)
+      return nodeOperatorsWithValidators.filter(
+        ({ id }) => !selectedIds.includes(id) || id === thisId,
+      )
+    }
+
+    const handleAddProgram = () =>
+      fieldsArr.append({
+        id: '',
+        vettedValidatorsLimit: '',
+      } as NodeOperator)
+
+    if (trustedCaller.initialLoading || isNodeOperatorsDataLoading) {
+      return <PageLoader />
+    }
+
+    if (trustedCaller.data !== walletAddress) {
+      return <MessageBox>You should be connected as trusted caller</MessageBox>
+    }
+
+    if (!nodeOperatorsList?.length || !nodeOperatorsWithValidators?.length) {
+      return (
+        <MessageBox>There are no node operators with validators yet</MessageBox>
+      )
+    }
+
+    return (
+      <>
+        {fieldsArr.fields.map((item, i) => {
+          return (
+            <Fragment key={item.id}>
+              <FieldsWrapper>
+                <FieldsHeader>
+                  {fieldsArr.fields.length > 1 && (
+                    <FieldsHeaderDesc>Update #{i + 1}</FieldsHeaderDesc>
+                  )}
+                  {fieldsArr.fields.length > 1 && (
+                    <RemoveItemButton onClick={() => fieldsArr.remove(i)}>
+                      Remove update {i + 1}
+                    </RemoveItemButton>
+                  )}
+                </FieldsHeader>
+
+                <Fieldset>
+                  <SelectControl
+                    label="Node operator"
+                    name={`${fieldNames.nodeOperators}.${i}.id`}
+                    rules={{ required: 'Field is required' }}
+                  >
+                    {getFilteredOptions(i).map(nodeOperator => (
+                      <Option
+                        key={nodeOperator.id}
+                        value={nodeOperator.id}
+                        children={`${nodeOperator.name} (id: ${nodeOperator.id})`}
+                      />
+                    ))}
+                  </SelectControl>
+                </Fieldset>
+
+                <Fieldset>
+                  <InputNumberControl
+                    label="Vetted validators limit"
+                    name={`${fieldNames.nodeOperators}.${i}.vettedValidatorsLimit`}
+                    rules={{
+                      required: 'Field is required',
+                      validate: value => {
+                        const check1 = validateToken(value)
```

Comment:
Why you use token validation for integer counter?

Reply:
You're right, I'll replace this validation with a proper one once https://github.com/lidofinance/easy-track-ui/pull/196 gets merged.